src/Main.java

```diff
@@ -0,0 +1,187 @@
+import address.Address;
+import configs.*;
+import currency.Currency;
+import customers.Company;
+import customers.Individual;
+import products.CurrentAccount;
+import services.ExchangeRateService;
+import tests.GeneralTest;
+import bank.Bank;
+
+import java.lang.Math;
+import java.time.LocalDate;
+
+public class Main {
+    public static void main(String[] args) {
+        // various tests during the development phase
+        boolean allowTests = true;
+        if (allowTests) {
+            GeneralTest.run();
```

Comment:
is it still useful for you at the moment? you should delete this from the main app where you call the functionality and if you want to create unit tests, you should have separate classes using Junit framework and during development phase you create tests in it that will exists after the dev phase as well in case something needs to be changed and you ensure your functionality is not broken after your changes through unit tests

Reply:
Am facut un nou push in care am eliminat partea de testing de care vorbiti.